modules/qdldl/0.1.7.bcr.1/source.json

```diff
@@ -0,0 +1,11 @@
+{
+    "url": "https://github.com/osqp/qdldl/archive/refs/tags/v0.1.7.tar.gz",
+    "integrity": "sha256-YxrmXzZ4Wfoe+t4WVuS6IrfaeJwG4BDM64spZWv2V1c=",
+    "strip_prefix": "qdldl-0.1.7",
+    "patches": {
+        "add_build_file.patch": "sha256-FSTwV1LSKSvI3a/l3hRx/5TGgbnn6CAawEk9tlH52W4=",
+        "add_cmake_configure_file.patch": "sha256-qge+NoLlQcrPkS5AR0TbJKkhE3FETgjuzvj9O0WqWwc=",
```

Comment:
FYI: If this file didn't change, you can reference this via `../../0.1.7/patches/X.patch`.